src/components/Panel/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Dialog, Transition } from "@headlessui/react";
+import React, { FC } from "react";
+
+const Panel: FC<any> = (props) => (
```

Comment:
@abbasali1994 @ap-justin I would suggest we align naming that we use for modals.

We call these "modals", in our code we have "ModalContext", "useModalContext", "show/closeModal", "KadoModal", "TxModal" etc, so we should keep this naming convention when referring to related components -> this component should be renamed `"Panel" -> "Modal"`.